api/v1beta1/common_types.go

```diff
@@ -150,6 +149,10 @@ type PodOptions struct {
 	// +listMapKey=whenUnsatisfiable
 	// +optional
 	TopologySpreadConstraints []corev1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
+
+	// ProvidedInitContainerResources is the resource requirements for the default init container(s) created by the Solr Operator if any are created.
```

Comment:
[0] Looks like this comment is out of sync with the new field-name you ultimately went with here?

Reply:
Good catch!